src/ring_theory/adjoin_root.lean

```diff
@@ -227,6 +227,13 @@ mul_div_eq_iff_is_root.2 $ is_root_root _
 
 end irreducible
 
+section is_noetherian_ring
+
+instance [comm_ring R] [is_noetherian_ring R] {f : R[X]} : is_noetherian_ring (adjoin_root f) :=
+by delta_instance adjoin_root
```

Comment:
It's tempting to replace this with the output of `show_term`, assuming this just unfolds to applying an instance somewhere else: that way, the reader can goto definition more easily.

Reply:
Yeah this is just ideal.quotient.is_noetherian_ring iirc, but this seemed more resilient to breakage though.

Reply:
It's easier to fix if it does break though.

Reply:
Easier to fix in what way? They break under quite different circumstances, if `delta_instance` breaks its presumably something very weird happening, and if adjoin_root is renamed then thats pretty clear what the fix is.

Reply:
@eric-wieser what do you think? This LGTM.

Reply:
Given the rest of the file uses `ideal.quotient.comm_ring` etc, I think it would make sense to match the style as:
```suggestion
ideal.quotient.is_noetherian_ring _
```